File: Fourth Semester/Systems-for-Design-and-Implementation/Lab03/Lab03-Java/src/domain/Participant.java
package domain;

public class Participant implements Identifiable<Integer>{
    private String nume;
    private Integer varsta;
    private String probe;

    public Participant(String nume, Integer varsta, String probe) {
        this.nume = nume;
        this.varsta = varsta;
        this.probe = probe;
    }

    public String getNume() {
        return nume;
    }

    public Integer getVarsta() {
        return varsta;
    }

    public String getProbe() {
        return probe;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    @Override
    public void setId(Integer integer) {

    }

    @Override
    public Integer getId() {
        return null;
    }
}

File: Fourth Semester/Systems-for-Design-and-Implementation/Lab03/Lab03-C#/Lab03/Lab03/domain/Participant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab03.domain
{
    public class Participant : Identifiable<int>
    {

        private String nume;

        private int varsta;

        private String probe;

        public Participant(String nume, int varsta, String probe)
        {
            this.nume = this.nume;
            this.varsta = this.varsta;
            this.probe = this.probe;
        }

        public String getNume()
        {
            return this.nume;
        }

        public int getVarsta()
        {
            return this.varsta;
        }

        public String getProbe()
        {
            return this.probe;
        }

        public void setNume(String nume)
        {
            this.nume = this.nume;
        }

      

        public void setId(int id)
        {
            throw new NotImplementedException();
        }

        int Identifiable<int>.getId()
        {
            throw new NotImplementedException();
        }
    }
   
}
